Styling approach #9

vyacheslav-pushkin opened the pull request.

vyacheslav-pushkin committed:
Styling approach #9

web-devel commented on src/style/palette.less:
```diff
@@ -0,0 +1,25 @@
+// NOTE: Use colors from antd/es/style/themes/default.less or antd/es/style/color/colors.less whenever possible
```
Let's take a look at https://ant.design/docs/spec/colors and @ant-design/colors

vyacheslav-pushkin committed:
Styling approach #9

Minor doc change:
 - removed reference to antd/es/style/color/colors.less as it is
   imported from antd/es/style/themes/default.less
